src/OrchardCore.Modules/OrchardCore.Forms/Views/Form.Wrapper.cshtml

```diff
@@ -0,0 +1,24 @@
+@using OrchardCore.ContentManagement
+@using OrchardCore.Forms.Models;
+@using OrchardCore.Mvc.Utilities;
+@using Newtonsoft.Json.Linq;
+
+@inject OrchardCore.ContentManagement.Display.IContentItemDisplayManager ContentItemDisplayManager
+
+@{
+    var formPart = ((ContentItem)Model.ContentItem).As<FormPart>();
+    var widgets = (JArray)((JObject)Model.ContentItem.Content)["FlowPart"]["Widgets"];
+    bool hasFileInput = false;
+
+    if(widgets.Count() > 0 )
+    {
+        hasFileInput = widgets?.Where( a=> a["InputPart"] != null && a["InputPart"]["Type"]?.Value<string>() == "file" )?.Count() > 0;   
+    }
+
+    string encType = hasFileInput ? "multipart/form-data" : "application/x-www-form-urlencoded";  
+}
+
+<form action="@formPart.Action" method="@formPart.Method"  enctype="@encType"  >
+    @Html.AntiForgeryToken()
```

Comment:
This should be an option. Either as settings or the part.

Reply:
moved to part